src/decode/lzma.rs

```diff
@@ -1,6 +1,7 @@
 use crate::decode::lzbuffer::{LzBuffer, LzCircularBuffer};
 use crate::decode::rangecoder;
 use crate::decode::rangecoder::RangeDecoder;
+use crate::decode::vec2d::Vec2D;
```

Comment:
It's not really a decode-specific code, so I'd put it in a `crate::util::` namespace.

Reply:
Moved to `crate::util::vec2d`